File: Code/Problem0006.py
import time
start_time = time.time()


sum_of_squares = 0
s = 0

for i in range(1, 101):
    sum_of_squares += (i)**2
    s += i
    
square_of_sum = (s)**2

print(square_of_sum - sum_of_squares)


end_time = time.time()
print(f'Program Execution Time: {end_time-start_time} seconds')

File: Code/Julia/Problem0006.jl
@time begin
function euler6()
    sum_of_squares = 0; s = 0
    for i in range(1, 100)
        sum_of_squares += (i)^2
        s += i
    end
    return (s)^2 - sum_of_squares
end
euler6()
end
print(euler6())